helm-charts/yunikorn/templates/ingress.yaml

```diff
@@ -0,0 +1,62 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if .Values.ingress.enabled -}}
+{{- $ingressPathType := .Values.ingress.pathType -}}
+{{- $svcPort := .Values.service.portWeb -}}
```

Comment:
Nit: `.Values.ingress.pathType` and `.Values.service.portWeb` are only used once each in this file, probably no need to have variables for them. On the other hand, `.Values.ingress.tls` is referred to more than once and yet no variable is created for it. I suggest just stick to using `.Values.<conf-name>` in all places. The rest of the PR LGTM

Reply:
I think we should only expose variables that are intended for the end user to change. Variables are not intended as shorthand for the templates.

Reply:
I reflect your review. Thanks for you review